src/Renci.SshNet/Channels/Channel.cs

```diff
@@ -158,7 +159,9 @@ public uint RemoteChannelNumber
             {
                 if (!_remoteChannelNumber.HasValue)
                 {
+#pragma warning disable S2372 // Exceptions should not be thrown from property getters
                     throw CreateRemoteChannelInfoNotAvailableException();
+#pragma warning restore S2372 // Exceptions should not be thrown from property getters
```

Comment:
You have disabled S2372 in the .editorconfig, so the suppressions should be unnecessary?